docs/overview.rst

```diff
@@ -380,7 +386,30 @@ The following formats are supported for JSON:
       {"id":1, "text": "example"}
       {"id":2, "text": "example2"}
 
+.. _overview-cluster-export:
+
+Export
+------
+
+The export tab allows users to download specific tables/views. When you first
+visit the Export tab, you can specify the name of a table/view, format (CSV,
+Parquet, JSON) and compression in which you'd like to export your data. 
```

Comment:
support for exporting to Parquet is being removed for the first iteration I believe